.env

```diff
@@ -1,13 +1,13 @@
-TESTVAR= {{ credentials.TESTVAR }}
-PORT = {{ credentials.PORT }}
-CRED_URL = {{ credentials.CRED_URL }}
-DID_URL = {{ credentials.DID_URL }}
-SCHEMA_URL = {{ credentials.SCHEMA_URL }}
-KEYCLOAK_REALM_ID= {{ credentials.KEYCLOAK_REALM_ID }}
-KEYCLOAK_CLIENT_ID= {{ credentials.KEYCLOAK_CLIENT_ID }}
-KEYCLOAK_CLIENT_SECRET= {{ credentials.KEYCLOAK_CLIENT_SECRET }}
-KEYCLOAK_URL= {{ credentials.KEYCLOAK_URL }}
-REGISTRY_URL= {{ credentials.REGISTRY_URL }}
-TELEMETRY_URL= {{ credentials.TELEMETRY_URL }}
+TESTVAR= development environment
```

Comment:
@YoginiTayade discard .env changes and then commit code

Reply:
resolved

Reply:
@YoginiTayade  .env file has issue
add below in that file and push code again
TESTVAR= {{ credentials.TESTVAR }}
PORT = {{ credentials.PORT }}
CRED_URL = {{ credentials.CRED_URL }}
DID_URL = {{ credentials.DID_URL }}
SCHEMA_URL = {{ credentials.SCHEMA_URL }}
KEYCLOAK_REALM_ID= {{ credentials.KEYCLOAK_REALM_ID }}
KEYCLOAK_CLIENT_ID= {{ credentials.KEYCLOAK_CLIENT_ID }}
KEYCLOAK_CLIENT_SECRET= {{ credentials.KEYCLOAK_CLIENT_SECRET }}
KEYCLOAK_URL= {{ credentials.KEYCLOAK_URL }}
REGISTRY_URL= {{ credentials.REGISTRY_URL }}
TELEMETRY_URL= {{ credentials.TELEMETRY_URL }}
VERIFICATION_URL= {{ credentials.VERIFICATION_URL }}
LEARNER_SCHEMA_FIELD= {{ credentials.LEARNER_SCHEMA_FIELD }}